run_bot_common.sh

```
@@ -0,0 +1,116 @@
+        fi
+
+        if [ -n "$STREAMING_AUDIO_FREQUENCY" ]; then
+            print_info "Injecting STREAMING_AUDIO_FREQUENCY from env: $STREAMING_AUDIO_FREQUENCY"
+            jq_cmd="$jq_cmd | .streaming_audio_frequency = ($STREAMING_AUDIO_FREQUENCY | tonumber)"
+        fi
+
+        echo "$config_json" | jq --arg bot_uuid "$bot_uuid" "$jq_cmd"
+    else
+        print_warning "jq not found, falling back to sed for bot_uuid (streaming params injection skipped)"
+        if echo "$config_json" | grep -q '"bot_uuid"[[:space:]]*:[[:space:]]*"[^\"]*"'; then
+            echo "$config_json" | sed 's/"bot_uuid"[[:space:]]*:[[:space:]]*"[^\"]*"/"bot_uuid": "'$bot_uuid'"/g'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)